### PR TITLE
Added bulk upload files

### PR DESCRIPTION
**Pull request by Cheshiriks**
- Added bulk upload files

Снимок

### CODE REVIEW

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt [hunk 1]
```diff
@@ -226,6 +226,48 @@ internal class OrganizationController(
...
+        organizationName
+    )
+        .flatMap {
+            organizationService.findByNameAndCreatedStatus(organizationName).toMono()
+        }
+        .switchIfEmptyToNotFound {
+            "No organization with name $organizationName was found."
+        }
+        .filter {
+            organizationPermissionEvaluator.hasGlobalRoleOrOrganizationRole(authentication, it.name, Role.SUPER_ADMIN)
```
Comment (sanyavertolet) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt:259]: Can be omitted here, as there is `@PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")`

**Review by sanyavertolet — APPROVED** [review]

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt]: Agree. Removed it

File: save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt [hunk 2]
```diff
@@ -239,7 +239,6 @@ internal class OrganizationController(
         Parameter(name = "isAbleToToBulkUpload", `in` = ParameterIn.QUERY, description = "new flag for bulk upload files ability", required = true),
     )
     @ApiResponse(responseCode = "200", description = "Successfully changed ability to bulk upload files.")
-    @ApiResponse(responseCode = "403", description = "Could not change ability to bulk upload files due to lack of permission.")
```
Comment (sanyavertolet) [inline save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt]: I guess `403` is returned if user is not `SUPER_ADMIN`

Comment (Cheshiriks) [reply save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/OrganizationController.kt]: Returned it

**Pull request merged**